### ISSUE TITLE
Teltonika Blue Puck RHT

### ISSUE THREAD

**Issue by ACrazyConcept** [custom-components/ble_monitor#721]
I had this laying around fo a while and just tried to add it.

I get this in the log with report_unknown: Teltonika

```
2022-02-17 16:58:44 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -71, MAC: F2:C5:EF:E5:C5:1C, DEVICE TYPE: None, ADV: 05166e2a5c0904166f2a29
2022-02-17 16:59:04 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -80, MAC: F2:C5:EF:E5:C5:1C, DEVICE TYPE: None, ADV: 05166e2a5b0904166f2a29
2022-02-17 16:59:14 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -75, MAC: F2:C5:EF:E5:C5:1C, DEVICE TYPE: None, ADV: 05166e2a5b0904166f2a29
2022-02-17 16:59:24 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -69, MAC: F2:C5:EF:E5:C5:1C, DEVICE TYPE: None, ADV: 05166e2a5a0904166f2a29
2022-02-17 16:59:34 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -76, MAC: F2:C5:EF:E5:C5:1C, DEVICE TYPE: None, ADV: 05166e2a580904166f2a2a
2022-02-17 16:59:34 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -59, MAC: F2:C5:EF:E5:C5:1C, DEVICE TYPE: None, ADV: 05166e2a580904166f2a2a
2022-02-17 16:59:44 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -69, MAC: F2:C5:EF:E5:C5:1C, DEVICE TYPE: None, ADV: 05166e2a550904166f2a29
2022-02-17 16:59:54 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -77, MAC: F2:C5:EF:E5:C5:1C, DEVICE TYPE: None, ADV: 05166e2a550904166f2a29
2022-02-17 17:00:04 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -68, MAC: F2:C5:EF:E5:C5:1C, DEVICE TYPE: None, ADV: 05166e2a540904166f2a29
2022-02-17 17:00:14 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -89, MAC: F2:C5:EF:E5:C5:1C, DEVICE TYPE: None, ADV: 05166e2a520904166f2a29
2022-02-17 17:00:14 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -59, MAC: F2:C5:EF:E5:C5:1C, DEVICE TYPE: None, ADV: 05166e2a520904166f2a29
2022-02-17 17:00:24 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -84, MAC: F2:C5:EF:E5:C5:1C, DEVICE TYPE: None, ADV: 05166e2a500904166f2a2a
2022-02-17 17:00:24 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -61, MAC: F2:C5:EF:E5:C5:1C, DEVICE TYPE: None, ADV: 05166e2a500904166f2a2a
2022-02-17 17:00:44 INFO (Thread-3) [custom_components.ble_monitor.ble_parser.teltonika] BLE ADV from UNKNOWN Teltonika DEVICE: RSSI: -59, MAC: F2:C5:EF:E5:C5:1C, DEVICE TYPE: None, ADV: 05166e2a4e0904166f2a29
```

And from btmon:

```
> HCI Event: LE Meta Event (0x3e) plen 40                                                                         #65 [hci0] 2.335326
      LE Advertising Report (0x02)
        Num reports: 1
        Event type: Connectable undirected - ADV_IND (0x00)
        Address type: Random (0x01)
        Address: F2:C5:EF:E5:C5:1C (Static)
        Data length: 28
        Flags: 0x06
          LE General Discoverable Mode
          BR/EDR Not Supported
        Service Data (UUID 0x2a6e): 530a
        Service Data (UUID 0x2a6f): 27
        Name (complete): P RHT 900DC8
        RSSI: -73 dBm (0xb7)
```

**Comment by Ernst79** [comment]
Looks like the `complete local name` is different than what we filter on

You will have to add the following on line 21 in teltonika.py. 

```
    elif complete_local_name[0:5] == "P RHT":
        device_type = "Blue Puck RHT"
```

I'll add it in a next release. 
```

**Comment by Ernst79** [comment]
Fixed in 7.8.1-beta

**Comment by ACrazyConcept** [comment]
> Fixed in 7.8.1-beta

It works. Thanks!

**Comment by Ernst79** [comment]
Fixed in 7.8.2 as final
